### DISCUSSION TITLE
How is the Node.js documentation generated?

### DISCUSSION THREAD

**Discussion by jhunterkohler** [nodejs/help#4847] [Q&A] [answered]
I understand that the hand written markdown documentation exists in the [docs]() folder. How do the project maintainers use that markdown to automatically generate the site documentation? As a novice developer, I would love to know how that documentation system works, or any tools it uses, because I am too captured by the prospect of having an _actually_ good documentation pipeline. 

Thanks so much if you have time to respond!

**Comment by targos** [comment, marked as answer]
You can find most of our documentation tooling in the [tools/doc](https://github.com/nodejs/node/tree/master/tools/doc) folder.
We use [remark](https://github.com/remarkjs/remark) to process the markdown files.

**Reply by jhunterkohler** [reply]
Thanks so much!!

**Comment by Trott** [comment]
Also see the docs-related tasks in `Makefile`.
